### PR TITLE
Enable functional tests on pull requests.

### PR DESCRIPTION
**Pull request by rhopp**
Do not merge yet.

### CODE REVIEW

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
[test]

**Comment by rhopp** [comment]
https://ci.centos.org/view/Devtools/job/devtools-rh-che-build-che-credentials-pr-check/

File: .ci/cico_run_functional_tests.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+echo "CHE VALIDATION: Running che-functional-tests"
+# git clone https://github.com/redhat-developer/che-functional-tests.git
+cp jenkins-env che-functional-tests/jenkins-env
+sed -i "s/export CHE_SERVER_DOCKER_IMAGE_TAG=.*/export CHE_SERVER_DOCKER_IMAGE_TAG=${CHE_SERVER_DOCKER_IMAGE_TAG}/g" che-functional-tests/config
+sed -i "s|export DOCKER_HUB_NAMESPACE=.*|export DOCKER_HUB_NAMESPACE=docker.io/rhchestage/che-server|g" che-functional-tests/config
+cd che-functional-tests
+export DO_NOT_REBASE=true
+. cico_run_EE_tests.sh
+echo "CHE VALIDATION: Che-functional tests passed."
```
Comment (benoitf) [inline .ci/cico_run_functional_tests.sh]: could you please add a newline at the end of the file ?

**Pull request merged**